### PR TITLE
OCI validation for unique employee email.

### PR DESCRIPTION
**Pull request by leanhdaovn**
OCI validation for unique employee email.
Copy validation code from keypay-integration.
https://trello.com/c/JrRtBGVe/1201-p1-oci-validation-for-unique-employee-email

### CODE REVIEW

**Comment by leanhdaovn** [comment]
@thukim please help review my code

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+machine:
+  ruby:
+    version: 2.2.0
```
Comment (thukim) [inline circle.yml]: if we dont run on circleCI, pls remove this file.

Comment (leanhdaovn) [inline circle.yml]: Isn't the file circle.yml for CircleCI only?

File: Gemfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
 # Specify your gem's dependencies in eh-myob.gemspec
-gem 'Rx.rb', git: 'https://github.com/ReactiveX/RxRuby.git', branch: 'master'
+gem 'Rx.rb', git: 'https://github.com/ReactiveX/RxRuby.git', branch: 'master', ref: 'e493b321a7a9157a15ff6239f2cbe45cf2d1efec'
```
Comment (thukim) [inline Gemfile]: is there any reason why you need to specify the ref version here?

File: eh-myob.gemspec [hunk 3]
```diff
@@ -19,7 +19,9 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.10"
+  spec.required_ruby_version = '2.2.0'
+
+  spec.add_development_dependency "bundler"
```
Comment (thukim) [inline eh-myob.gemspec]: wondering why you skipped `"~> 1.10"` for `bundler`

File: circle.yml (hunk 1 above)
Comment (thukim) [inline circle.yml]: yes, it is for CircleCI only

**Comment by thukim** [comment]
Reviewed 👍

File: Gemfile (hunk 2 above)
Comment (leanhdaovn) [inline Gemfile]: I discussed this problem with @gaugau 
The latest master branch of that Repo breaks our code.
I copied the ref from Gemfile.lock from our main project.

File: eh-myob.gemspec (hunk 3 above)
Comment (leanhdaovn) [inline eh-myob.gemspec]: b/c I saw this error message from CircleCI

**Pull request merged**